dspace-api/src/test/data/dspaceFolder/config/local.cfg

```diff
@@ -25,6 +25,9 @@
 # SERVER CONFIGURATION   #
 ##########################
 
+# Spring boot test by default mock the server on the localhost (80)
+dspace.baseUrl = http://localhost
```

Comment:
this is required as Spring Boot Test will mock our service as if it was deployed as ROOT application on port 80